Ruby/Ruby.sublime-syntax

```diff
@@ -745,6 +745,7 @@ contexts:
 
   method-parameters:
     - meta_content_scope: meta.function.parameters.ruby
+    - include: comments
```

Comment:
Ah, because of the comma on line 754 I think. If you put it above there, it'll work I think.